recipe/conda_build_config.yaml

```
@@ -4,6 +4,17 @@ c_stdlib_version:  # [unix]
 MACOSX_SDK_VERSION:         # [osx]
   - 13.3                    # [osx]
 
+channel_targets:
+  - conda-forge main
+  - conda-forge pytorch_rc
+rc:
+  - None
+  - rc
```

Please change this to
```suggestion
is_rc:
  - False
  - True
```